### PR TITLE
Amended Landscape content to encompass recent updates to Landscape.

### PR DESCRIPTION
**Pull request by alysonrose88**

### CODE REVIEW

File: landscape.rst [hunk 1]
```diff
@@ -31,9 +31,26 @@ There are 3 versions of Landscape:
...
 
+Landscape SaaS
+~~~~~~~~~~~~~~
 Before you can access your Landscape SaaS account for the first time, you need to follow the steps under :doc:`Account set-up <account-setup>`.
 
-For customers using self-hosted Landscape, your ``licence.txt`` file is available in your Landscape SaaS account. You will need to re-download and apply new ``licence.txt`` files every time you purchase new Ubuntu Pro licences and after every renewal.
+Self-hosted Landscape
+~~~~~~~~~~~~~~~~~~~~~
+
+We currently have two licensing mechanisms for self hosted Landscape: the new, Pro client method and the old, licence.txt file method.
```
Comment (ingarisle) [inline landscape.rst:41]: Change adding link (as discussed)
After you have set up self-hosted Landscape server (hyperlink to Landscape docs) there are two licensing mechanisms: the new, Pro client method and the old, licence.txt file method.

File: account-setup.rst [hunk 2]
```diff
@@ -85,9 +85,11 @@ If a secondary user is not already listed, add one now with the *New user* butto
 Landscape
 ---------
 
-You will have received an email message inviting you to activate your Landscape account (Subject: "You have been invited to the Landscape account […]", from ``noreply+landscape@canonical.com``). 
+If you have a paid Ubuntu Pro subscription and would like to use Landscape SaaS, contact your Account Manager or Canonical’s Customer Success team to request an invitation to join your Landscape account.
```
Comment (ingarisle) [inline account-setup.rst:88]: add cross linke to "Self-hosted landscape" in understanding landscape section to address self hosted landscape

**Review by abbeyadejumo — APPROVED** [review]

File: account-setup.rst [hunk 3]
```diff
@@ -85,13 +85,11 @@ If a secondary user is not already listed, add one now with the *New user* butto
 Landscape
 ---------
 
-If you have a paid Ubuntu Pro subscription and would like to use Landscape SaaS, contact your Account Manager or Canonical’s Customer Success team to request an invitation to join your Landscape account.
+If you have a paid Ubuntu Pro subscription and would like to use Landscape SaaS, contact your Account Manager or Canonical’s Customer Success team to request an email invitation to join your Landscape account. Then accept the invitation and follow the instructions.
 
 You can find the contact details for Customer Success in the original “Welcome to Ubuntu Pro” message we sent you. If you cannot find the welcome message, go to https://ubuntu.com/ and ask for Customer Success contact details using the LiveChat service.
 
-Then accept the invitation and follow the instructions (if you can't find the message, check your spam filter, and then ask us).
```
Comment (ingarisle) [inline account-setup.rst]: I think we should leave the bit about (if you can't find the message, check your spam filter, and then ask us) in as well as the address the message will originate from like we have in the current version. Suggestion: add in line 88 "...to request an email invitation to join your Landscape account. The email will be sent from noreply+landscape@canonical.com, if you cannot find the message, check your spam filter and then ask us. Accept the invitation and follow the instructions.

**Review by ingarisle — APPROVED** [review]
looks great!

**Pull request merged**